packages/modular-scripts/src/react-scripts/scripts/startWebpack.ts

```diff
@@ -0,0 +1,150 @@
+import fs from 'fs';
+import chalk from 'chalk';
+import webpack from 'webpack';
+import WebpackDevServer from 'webpack-dev-server';
+import { clearConsole, log } from '../../react-dev-utils/logger';
+import {
+  choosePort,
+  createCompiler,
+  prepareProxy,
+  prepareUrls,
+} from '../../react-dev-utils/WebpackDevServerUtils';
+import openBrowser from '../../react-dev-utils/openBrowser';
+import isCI from 'is-ci';
+
+import getConfig from '../config/webpack.config';
+import createDevServerConfig from '../config/webpackDevServer.config';
+import { Paths } from '../../utils/determineTargetPaths';
+
+// Makes the script crash on unhandled rejections instead of silently
+// ignoring them. In the future, promise rejections that are not handled will
+// terminate the Node.js process with a non-zero exit code.
+process.on('unhandledRejection', (err) => {
```

Comment:
Does this still make sense now that we're not running it as a separate JS script anymore